src/main/java/frc/robot/RobotContainer.java

```diff
@@ -48,7 +48,8 @@ public RobotContainer() {
         coralSubsystem = new CoralSubsystem();
         algaeSubsystem = new AlgaeSubsystem();
         lift = new LiftSubsystem(driveSubsystem.chassis::SetAccelerationLimit, coralSubsystem, algaeSubsystem);
-        lift.setDefaultCommand(new LiftPidControl(lift, () -> payloadController.getLeftY()));
+        lift.setDefaultCommand(new LiftPidControl(lift, () -> payloadController.getLeftY(),
+                payloadController.zl().getAsBoolean() && payloadController.zr().getAsBoolean()));
```

Comment:
Because it's a boolean being evaluated here, rather than a boolean supplier, the LifdPidControl command will run ````payloadController.zl().getAsBoolean() && payloadController.zr().getAsBoolean())````, store it as its value at the time RobotContainer is constructed (which will almost always be ````false```), then create the LiftPidControl command and pass the ````false```` or ```true```` in, and then never update it later.